var/spack/repos/builtin/packages/openblas/package.py

```diff
@@ -201,7 +201,7 @@ def make_defs(self):
         if '+ilp64' in self.spec:
             make_defs += ['INTERFACE64=1']
 
-        if 'x86' in self.spec.architecture.target.lower():
+        if 'x86' in str(self.spec.architecture.target).lower():
```

Comment:
Same comment as in #12914. Also (not a review request, just a question): would you like to rework openblas similarly to what is done for `fftw` in #12889?

Reply:
I'm not  familiar with the openblas package, just needed it to compile fine since it's needed by py-numpy. I'll leave any further modifications to someone else.